.editorconfig

```diff
@@ -1,7 +1,7 @@
 root = true
 
 [*]
-max_line_length = 300
+#max_line_length = 300
```

Comment:
?

Reply:
we unfortunately don't obey this nearly anywhere. there were so many failures

Reply:
So, remove it, increase it, add a TODO comment?

For content, we shouldn't have this rule anyway. It's fine to write paragraphs on one line. In fact, I would strongly prefer that with text editing. I do think we need it for code files in this repository.